### DISCUSSION TITLE
Feedback regarding the latest update

### DISCUSSION THREAD

**Discussion by Shuro** [mbround18/valheim-docker#65] [Ideas]
Hey Michael, I've seen that you added libsdl2-dev instead of libsdn2.0-0.0. Is that necessary? I'm pretty sure there is nothing to be build with it, so not sure if all the development additions are necessary. Also a quick reminder that the apt cache should still be cleared with `&& rm -rf /var/lib/apt/lists/* ` after package install is done.

**Comment by mbround18** [comment]
That package is not found with apt-get, any advice?

![image](https://user-images.githubusercontent.com/12646562/107835844-b9ae8b80-6d4f-11eb-8acc-83632e6de560.png)

**Reply by Shuro** [reply]
My mistake, it is libsdl, not libsdn.

```
shuro@mainframe:~/games$ docker-compose exec valheim /bin/bash
root@40de2e0ce892:/home/steam/steamcmd# apt-cache search libsdl2
libsdl2-2.0-0 - Simple DirectMedia Layer
[..]
```

**Reply by mbround18** [reply]
Will add this to a branch a tomorrow <3

**Comment by mbround18** [comment]
Got a PR up for this now :) #77
